src/main/java/com/elixr/poc/bulkimport/service/DoctorService.java

```diff
@@ -0,0 +1,18 @@
+package com.elixr.poc.bulkimport.service;
+
+import com.elixr.poc.bulkimport.dto.Doctor;
+import com.elixr.poc.bulkimport.repository.DoctorRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DoctorService {
+    private final DoctorRepository doctorRepository;
+
+    public DoctorService(DoctorRepository doctorRepository) {
+        this.doctorRepository = doctorRepository;
+    }
+
+    public void createNewDoctor(Doctor doctor) {
+        doctorRepository.save(doctor);
```

Comment:
Based on my previous comment, you have to build the Doctor object from rest/data/Doctor object here if you are taking cognizance of my previous comment

Reply:
Updated.